components/card/index.css

```diff
@@ -10,40 +10,168 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+  /* TODO: These are placeholder until recursive RGB is available */
+  .spectrum--light &,
+  .spectrum--lightest & {
+    --spectrum-card-selected-background-color-rgb: var(--spectrum-blue-900-rgb);
+  }
+  .spectrum--dark & {
+    --spectrum-card-selected-background-color-rgb: var(--spectrum-blue-500-rgb);
+  }
+  .spectrum--darkest & {
+    --spectrum-card-selected-background-color-rgb: var(--spectrum-blue-600-rgb);
+  }
 
-.spectrum-Card--sizeS {
-  @remapvars {
-    find: /--spectrum-card(.*)-s-/;
-    filter: color;
-    replace: --spectrum-card$1-;
+  /* Quiet/Gallery */
+  &--quiet,
+  &--gallery {
+    --mod-card-content-margin-top: var(--spectrum-card-content-margin-top--quiet, var(--spectrum-spacing-100));
+    --mod-card-minimum-width: var(--spectrum-card-minimum-width--quiet, var(--spectrum-card-minimum-width));
+
+    --spectrum-card-preview-border-width: var(--spectrum-border-width-100);
   }
-}
 
-.spectrum-Card--sizeM {
-  @remapvars {
-    find: /--spectrum-card(.*)-m-/;
-    filter: color;
-    replace: --spectrum-card$1-;
+  &--quiet,
+  &--gallery,
+  &--horizontal {
+    --mod-card-background-color: var(--spectrum-card-background-color--quiet, var(--spectrum-background-base-color));
+    --spectrum-card-background-color-hover: var(--spectrum-card-background-color-hover--quiet, var(--spectrum-gray-300));
   }
+
+  /* Horizontal */
+  --spectrum-card-horizontal-body-padding: var(--spectrum-spacing-300);
+  --spectrum-card-horizontal-preview-padding: var(--spectrum-spacing-200);
 }
 
 .spectrum-Card {
   position: relative;
-
   display: inline-flex;
   flex-direction: column;
-
   box-sizing: border-box;
-  min-inline-size: var(--spectrum-card-min-width); /* setting min-width incase a product overrides width */
+  text-decoration: none;
 
-  border: var(--spectrum-card-border-size) solid transparent;
-  border-radius: var(--spectrum-card-border-radius);
+  /* TODO: This should use the --spectrum-card-default-width token when added */
+  inline-size: 200px;
+  min-inline-size: var(--mod-card-minimum-width, var(--spectrum-card-minimum-width));
 
-  text-decoration: none;
+  border: var(--mod-card-border-width, var(--spectrum-card-border-width)) solid transparent;
+  border-radius: var(--mod-card-corner-radius, var(--spectrum-card-corner-radius));
+  border-color: var(--highcontrast-card-border-color, var(--mod-card-border-color, var(--spectrum-card-border-color)));
+  background-color: var(--highcontrast-card-background-color, var(--mod-spectrum-card-background-color, var(--spectrum-card-background-color)));
+
+  &::before {
+    content: '';
+    block-size: 100%;
+    inline-size: 100%;
+    position: absolute;
+    inset-inline-start: 0;
+    inset-block-start: 0;
+  }
 
-  &:focus {
+  /* Focus indicator - use after so change in border-width does not affect card width */
+  &::after {
+    content: '';
```

Comment:
I feel like I've seen t his set of styles on other components - I might noodle on if a placeholder would be helpful for this in future. Maybe post-build rewrite